src/Content/JsonContent.cs

```diff
@@ -0,0 +1,59 @@
+﻿// <copyright file="JsonContent.cs" company="ProjectBlackmagic">
+// RestfulClient
+// Copyright (c) ProjectBlackmagic. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Github: https://github.com/ProjectBlackmagic/RestfulClient.
+// </copyright>
+
+using System;
+using System.Net.Http;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ProjectBlackmagic.RestfulClient.Content
+{
+    /// <summary>
+    /// JsonContent defines a subtype of <see cref="StringContent"/> for JSON content.
+    /// </summary>
+    /// <typeparam name="T">Type of object being serialized</typeparam>
+    public class JsonContent<T> : StringContent
```

Comment:
This took me about 20 minutes to figure out how it works. I think it might be simplified and made more clear, but this will do just fine as well.

Reply:
I'm open to suggestions